app/assets/javascripts/modules/project.js

```diff
@@ -30,4 +30,45 @@ angular.module('probedock.project', [ 'probedock.api', 'probedock.forms', 'probe
     });
   })
 
+  .directive('projectRecentActivity', function() {
+    return {
+      restrict: 'E',
+      controller: 'ProjectRecentActivityCtrl',
+      controllerAs: 'ctrl',
+      templateUrl: '/templates/project-recent-activity.html',
+      scope: {
+        organization: '=',
+        project: '='
+      }
+    };
+  })
+
+  .controller('ProjectRecentActivityCtrl', function(api, $scope) {
+
+    $scope.$watch('project', function(value) {
+      if (value) {
+        fetchReports();
+      }
+    });
+
+    function fetchReports() {
+      return api({
+        url: '/reports',
+        params: {
+          pageSize: 5,
+          organizationId: $scope.organization.id,
+          projectId: $scope.project.id,
+          withRunners: 1,
+          withProjects: 1,
+          withProjectVersions: 1,
+          withCategories: 1
```

Comment:
You should add the `withProjectCountsFor: $scope.project.id` option, that way you'll be able to retrieve the number of new tests specific to that project, not the report (which may contain multiple projects).

It could also be possible to display the passed/failed counts of the project instead of the report, but I would not do that change now, as it would then be inconsistent with the categories/tags, etc.